elastic/kibana
# Feature/remove asset criticality flag

## Summary

It removes the asset criticality advanced setting, which enables the feature by default for all users.

Deleted settings:
![Screenshot 2024-10-15 at 14 54 48](https://github.com/user-attachments/assets/103c3f04-fd7e-45cf-ac74-93e1eef341fa)

### How to test it?
* Start Kibana with security data
* Inside security solution / manage, you should be able to find the Asset Criticality page
![Screenshot 2024-10-15 at 14 57 14](https://github.com/user-attachments/assets/7ddcee91-ad76-4d8f-b14a-bacc4ba31172)
* You should see the asset critically section when opening an entity flyout (explore or host page) <img width="400" src="https://github.com/user-attachments/assets/3a9ee545-566c-4687-af16-f31bd93bdc20" />
* The risk score should be updated if you update an entity's asset criticality.



### Checklist
